Nataly/src/pages/capital-page/capitalPage.page.tsx

```diff
@@ -0,0 +1,195 @@
+import React  from "react"
+
+import Header from "../../components/header/header.component"
+import PreviewMovies from "../../components/previewMovies/previewMovies.component"
+import SearchBlock from "../../components/searchBlock/searchBlock.component"
+import Footer from "../../components/footer/footer.component"
+import ErrorEmptyResults from "../../components/error-empty-results/error-empty-results.component"
+
+import {sections} from "../../directory.data"
+
+import UserSelectedMovies from "../../components/user-selected-movies/userSelectedMovies.component"
+import SortDetails from "../../components/sort-details/sort-details.component"
+import { sortAndDeduplicateDiagnostics } from "typescript"
+
+
+interface ICapitalPageState {
+    value: string,
+    hidden: boolean,
+
+    сolorActive: boolean, 
+
+    colorActiveSort: boolean,
+
+    titleOrGenre: boolean,
+
+    rating: boolean,
+    release_data: boolean,
+
+    number_of_films: number,
+
+    moviesDefault: any // [] дает ошибку!!!!
+    
+    movies: any,
+
+    renderResult: boolean,
+
+    handleChangSortValue?: any,
+}
+
+const rest = fetch('http://reactjs-cdp.herokuapp.com/movies')
+.then(res => res.json())
+.then(res => console.log(res.data))
+
+class CapitalPage extends React.Component<{}, ICapitalPageState> {
+    
+    data: any = sections.data 
+
+    state: ICapitalPageState = {
+        value: "",
+        hidden: false,
+        сolorActive: true, 
+        colorActiveSort: true,
+        titleOrGenre: true,
+        rating: true,
+        release_data: false,
+        number_of_films: 0,
+
+        moviesDefault: [],
+        movies: [],
+
+        renderResult: false,
+    }
+
+    componentDidMount(){
+        setTimeout(() => {
+            fetch('http://reactjs-cdp.herokuapp.com/movies')
+            .then(res => res.json())
+            .then(res => 
+                this.setState({moviesDefault: res.data}))
+        }, 1000)
+        
+    }
+
+    
+
+    handleClickToggle = (e: React.MouseEvent) => {
+        e.preventDefault()
+        console.log("works")
+        this.setState({
+            titleOrGenre: !this.state.titleOrGenre,
+            сolorActive: !this.state.сolorActive
```

Comment:
You don't need state to apply right color. you can handle it inside of component. just compare like this 
`value === 'genre' ? '.active' : ''`